app/src/org/commcare/android/database/app/AppDatabaseUpgrader.java

```diff
@@ -40,6 +41,9 @@ public void upgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                 oldVersion = 5;
             }
         }
+        if (oldVersion == 5) {
+            upgradeFiveSix(db);
```

Comment:
@ctsims `oldVersion` is not referenced after its assignment in the cases above. We should use comments if it just used to notify readers of the code. We could also choose to log the versions.